lib/matplotlib/backends/backend_ps.py

```diff
@@ -1440,27 +1440,32 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
```

Comment:
Oops, I meant to say, do the same thing as `texmanager.py`; `precmd` is just setting the drive letter, which is the same thing that `_get_shell_cmd` did there, and it seems to work on AppVeyor.